src/controllers/trainee/Controller.ts

```diff
@@ -1,77 +1,88 @@
-            }
-      }
-      delete (req, res, next) {
-            try {
-                  console.log('Inside delete method of trainee controller');
-                  res.send( {
-                        message: 'Trainee deleted successfully',
-                        data: [
-                              {
-                                    name: 'Trainee1',
-                                    address: 'Delhi, IN'
-                              }
-                        ]
-                  });
-            } catch (err) {
-                  console.log('Inside err', err);
-            }
-      }
+    get(req, res, next) {
+        try {
+            console.log('Inside GET method of Trianee controller ');
+            userRepository.getAll()
+                .then((res1) => {
+                    console.log('Response is: ', res1);
+                    res.status(200).send({ message: 'successfully fetched Trainee', data: res1 })
+                })
+        } catch (err) {
+            console.log('Inside Error', err);
+        }
+    }
+    create(req, res, next) {
+        try {
+            const { role, name, email, password } = req.body
+            console.log('Inside POST method of Trianee controller ');
+            userRepository.create({ role, name, email, password })
+                .then((res1) => {
+                    console.log('Response is: ', res1);
+                    res.status(200).send({ message: 'Trainee created successfully', data: res1 })
+                })
+        } catch (err) {
+            console.log('Inside Error', err);
+        }
+    }
+    update(req, res, next) {
+        try {
+            const { role, name, id, email, password } = req.body;
+            console.log('Inside Update method of Trianee controller ', id);
+            userRepository.findOne({ originalId: id })
+                .then((data) => {
+                    if (data != null) {
+                        userRepository.update(id, { updatedAt: Date.now(), updatedBy: id, createdBy: id, deletedAt: Date.now(), deletedBy: id, name: name || data.name, role: role || data.role, email: email || data.email, password: password || data.password })
+                            .then((data) => {
+                                console.log("respnse is ", data);
+                                res.status(200).send({ message: "successfully update", data: data });
+                            })
+                    }
+                });
+        } catch (err) {
+            console.log('Inside Error', err);
+        }
+    }
+    public delete(req, res, next) {
+        try {
+            const id = req.params.id;
+            const userData = userRepository.findOne({ originalId: id })
+            userRepository.findOne({ originalId: id })
+            console.log(id, "  Value of ID")
+            const remover = '5fb363da080091a8c21d58b';
+            console.log(remover, " remover")
+            const user = new UserRepository();
```

Comment:
remove const user = new UserRepository();
this is already defined in the global scope use userRepository